Fix typos and small issues with the tutorial (#125)
ESMValGroup/ESMValTool_Tutorial: _episodes/02-installation.md

    should work on any recent system. If you have problems in the next step(s)

>     you can alternatively try a 32 bit version.

Reviewer: Please replace all of the text before the code block below with "Please download Miniconda3 by running"

[`julia-1.0.5-linux-x86_64.tar.gz`](https://julialang-s3.julialang.org/bin/linux/x64/1.0/julia-1.0.5-linux-x86_64.tar.gz)
by clicking the link or going to the [Julia downloads

> page](https://julialang.org/downloads/). Download the file directly to the

Reviewer: Please remove this paragraph, no need to have three different instructions for downloading the file.

> `~/julia` directory or move it there after downloading. To download and extract the file, you can use the following commands:

Reviewer: ```suggestion
Next, to download and extract the file `julia-1.0.5-linux-x86_64.tar.gz`, you can use the following commands:
```
